ansible/system.yml

```diff
@@ -57,54 +64,56 @@
   become: yes
   roles:
     - role: postgresql
-      when: esrender_db_type == 'postgresql'
+      when: esrender_db_type == 'postgresql' and edu_version is version('7.0.0','<')
     - role: mariadb
-      when: esrender_db_type == 'mariadb'
+      when: esrender_db_type == 'mariadb' and edu_version is version('7.0.0','<')
     - renderingservice-installation
     - role: splash
-      when: install_splash is defined and install_splash
+      when: install_splash is defined and install_splash and edu_version is version('7.0.0','<')
   tags:
     - root-task
 
 - hosts: opencast
   roles:
     - role: opencast-import
-      when: install_opencast_importer is defined and install_opencast_importer
+      when: install_opencast_importer is defined and install_opencast_importer and edu_version is version('7.0.0','<')
   tags:
     - opencast
 
 - hosts: renderingservice
   roles:
-    - renderingservice-init
-    - renderingservice-registration
+    - role: renderingservice-init
+      when: edu_version is defined and edu_version is version('7.0.0','<')
+    - role: renderingservice-registration
+      when: edu_version is defined and edu_version is version('7.0.0','<')
 
 - hosts: onlyoffice
   become: yes
   roles:
     - role: postgresql
-      when: enable_only_office is defined and enable_only_office
+      when: enable_only_office is defined and enable_only_office and edu_version is version('7.0.0','<')
     - role: geerlingguy.rabbitmq
-      when: enable_only_office is defined and enable_only_office
+      when: enable_only_office is defined and enable_only_office and edu_version is version('7.0.0','<')
     - role: nginx
-      when: enable_only_office is defined and enable_only_office
+      when: enable_only_office is defined and enable_only_office and edu_version is version('7.0.0','<')
     - role: onlyoffice
-      when: enable_only_office is defined and enable_only_office
+      when: enable_only_office is defined and enable_only_office and edu_version is version('7.0.0','<')
 
 - hosts: edusharing
   roles:
     - role: moodle-registration
-      when: register_moodle
+      when: register_moodle and edu_version is version('7.0.0','<')
   tags:
     - moodle-registration
-    
+
 - hosts: edusharing
   become: yes
   roles:
     - role: antivirus
-      when: install_antivirus is defined and install_antivirus
+      when: install_antivirus is defined and install_antivirus and edu_version is version('7.0.0','<')
       tags: antivirus
     - role: shibboleth
-      when: install_shibboleth is defined and install_shibboleth
+      when: install_shibboleth is defined and install_shibboleth and edu_version is version('7.0.0','<')
```

Comment:
what about the shibboleth-integration? Just don't install it from 7.0.0 is not a good option

Reply:
still open